transport-protocol/src/custom.rs

```diff
@@ -14,8 +14,6 @@ use crate::transport_protocol;
 use core::convert::TryInto;
 use err_derive::Error;
 use protobuf::{error::ProtobufError, Message, ProtobufEnum};
-#[cfg(feature = "sgx_attestation")]
-use sgx_types;
```

Comment:
If there is a CI problem in sgx, this line might need to stay.

Reply:
The SGX build seems to complete, and (from memory) all of the SGX tests passed, too, though I will double check.